linter/allowlists/de.txt

```diff
@@ -511,6 +511,7 @@ Themenwelten
 Ticker
 Tiles
 Titelleiste
+title
```

Comment:
The linter does not report `title` as an error but `WebP`. Could you add `WebP` to the `default.txt` allowlist instead of `title`?

![image](https://user-images.githubusercontent.com/1218766/147687236-c5cb82a5-4afc-42af-bdce-5cba976d6d79.png)